task/01-strings-tasks.js

```diff
@@ -221,7 +253,17 @@ function getRectangleString(width, height) {
  *
  */
 function encodeToRot13(str) {
-    throw new Error('Not implemented');
+    var encoded = '';
+    for (var i = 0; i < str.length; i++) {
+        encoded += encode(str[i]);
+    }
+    return encoded;
+
+    function encode(c) {
+        var input = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+        var output = 'NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm';
+        return input.indexOf(c) === -1 ? c : output[input.indexOf(c)];
+    }
 }
```

Comment:
`var` is deprecated. It has no sense to use for browsers code even currently.

Reply:
ok i got it